src/utils/utils.go

```diff
@@ -247,3 +248,52 @@ func IsVersionLessThan47(openshiftVersion string) (bool, error) {
 func EtcdPatchRequired(openshiftVersion string) (bool, error) {
 	return IsVersionLessThan47(openshiftVersion)
 }
+
+func CsvStatusToOperatorStatus(csvStatus string) models.OperatorStatus {
+	switch csvStatus {
+	case "Succeeded":
+		return models.OperatorStatusAvailable
+	case "Failed":
+		return models.OperatorStatusFailed
+	default:
+		return models.OperatorStatusProgressing
+	}
+}
```

Comment:
Can we use CSVPhase constants? 
https://github.com/operator-framework/api/blob/718d279192d30713e6e07a8833a256d27e0c33d4/pkg/operators/v1alpha1/clusterserviceversion_types.go#L327-L348